FAQ.md

```diff
@@ -237,7 +237,8 @@ The local training is the most hardware intensive part of this process. Your har
 
 To run and train InstructLab locally, you must meet the following requirements:
 
-- A Linux-based operating system OR an Apple Silicon M1, M2, or M3 system
+- A Linux-based operating system OR an Apple Silicon M1, M2, or M3 system OR a
+  Windows system with WSL
```

Comment:
WSL is an acronym which should be expanded. 

Since we have 3 things, maybe subbullets?

Reply:
addressed both items, thanks!